annotation/tests/test_tasks_services.py

```diff
@@ -0,0 +1,322 @@
+from typing import Dict, List, Union
+from unittest.mock import Mock, call, patch
+
+import pytest
+from fastapi import HTTPException
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import ValidationSchema
+from annotation.models import File, ManualAnnotationTask
+from annotation.tasks.services import (
+    check_cross_annotating_pages,
+    validate_files_info,
+    validate_task_info,
+    validate_user_actions,
+    validate_users_info,
+)
+
+
+@patch("sqlalchemy.orm.Session", spec=True)
```

Comment:
You can simply use it inside the test as a patch - as you do with other patches

Reply:
Done